bazel/BUILD.bazel

```diff
@@ -9,6 +9,7 @@ filegroup(
         "//bazel/browsers:files",
         "//bazel/constraints:files",
         "//bazel/esbuild:files",
+        "//bazel/http-server:files",
```

Comment:
Good question. I renamed it to something less scoped, because technically this could also be used in e2e tests etc. "devserver" sounded like something I would only use for the dev-app (really subjective feeling though 😄 )

Reply:
I'll continue with that, as I think it's better subjectively. Totally happy to re-consider/rename if you have other feelings.